org.eclipse.buildship.ui.test/src/main/groovy/org/eclipse/buildship/ui/SWTBotTestHelper.groovy

```diff
@@ -0,0 +1,94 @@
+package org.eclipse.buildship.ui
+
+import org.eclipse.swtbot.eclipse.finder.SWTWorkbenchBot
+import org.eclipse.swtbot.eclipse.finder.widgets.SWTBotView
+import org.eclipse.swtbot.swt.finder.exceptions.WidgetNotFoundException
+import org.eclipse.swtbot.swt.finder.widgets.SWTBotShell
+import org.eclipse.swtbot.swt.finder.finders.UIThreadRunnable
+import org.eclipse.swtbot.swt.finder.results.VoidResult
+import org.eclipse.swtbot.swt.finder.results.BoolResult
+import org.eclipse.ui.PlatformUI
+import org.eclipse.core.runtime.jobs.Job
+import org.eclipse.swt.widgets.Display
+
+class SWTBotTestHelper {
```

Comment:
This class is a bit weird. It's not entirely meant to be a singleton. The _BaseSWTBotTest_ class was inherited by the Buildship test cases, which means that both the `BaseSWTBotTest` and its child could share the same `SWTWorkbenchBot`. This approach delegates all bot control to the `SWTBotTestHelper` class, and uses only static methods, much like our implementation in the [Smart Import](https://github.com/eclipse/buildship/pull/122) feature.

This class is no doubt weird, though, and is something that I pointed out in my [request for review](https://github.com/Ian-Stewart-Binks/buildship/pull/7).